scientific_library/tvb/simulator/models/jansen_rit.py

```diff
@@ -459,6 +459,7 @@ class ZetterbergJansen(Model):
         it is also provides the default range of phase-plane plots.""")
 
     variables_of_interest = List(
+        of=str,
```

Comment:
These are changes ok to be made even if they are in the library. 
They are left-ver bugs from our neotraits migration.